WordPress/src/main/java/org/wordpress/android/ui/posts/PostResolutionOverlayListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.wordpress.android.ui.posts
+
+interface PostResolutionOverlayListener {
+    fun onOverlayAction(action: PostResolutionOverlayAction)
```

Comment:
Another minor suggestion here. Maybe we could rename this to `onPostResolutionAction()` for clarity. This will be implemented in the activity so if someone sees a function there named `onOverlayAction()` might not realise what it actually does.

Reply:
Agree, and it also is on my list of refactoring todos